src/components/routes.tsx

```diff
@@ -141,7 +141,10 @@ export const Routes: React.FC = () => {
       <Route path={ScreenPaths.USERNAME} element={<Username />} />
       {/*Sign In*/}
       <Route path={ScreenPaths.SIGN_IN} element={getSignInComponent()} />
-      <Route path={ScreenPaths.RECOVERY_CODE} element={<DecryptRecoveryCode />} />
+      <Route
+        path={ScreenPaths.RECOVERY_CODE}
+        element={<MagicRecoveryCode key="MagicRecoveryCode" />}
```

Comment:
What is the `key` for?

Reply:
was debugging the input bug, passing a key forces a render and prevents sharing of some components

Reply:
but this did not fix it, so i can remove